Docs/Parameters XML File/Parameters_XML_Schema.xsd

```diff
@@ -14,7 +21,11 @@
                   </xs:simpleType>
                </xs:element>
                <xs:element name="terminals" minOccurs="0" maxOccurs="1">
-                  <xs:complexType>
+               <xs:unique name="onlyOnebusController">
+                  <xs:selector xpath="terminal" />
+                  <xs:field xpath="busController" />
+               </xs:unique>
+				  <xs:complexType>
```

Comment:
It looks like the problem is that some lines have indents instead of spaces like most of the rest. I would just replace the tabs with spaces, then true them up with the existing content.
![image](https://user-images.githubusercontent.com/31290917/226461488-093df394-2390-40ec-81eb-54a39b69a81d.png)

Reply:
fixed in bdfff68de31979898f99c75bd02386861a507680